src/Config/ConfigurationStore.php

```diff
@@ -15,6 +17,7 @@ class ConfigurationStore implements IConfigurationStore
     private CacheInterface $metadataCache;
 
     const FLAG_TIMESTAMP = "flagTimestamp";
+    const BANDIT_VARIATION_KEY = 'banditVariations';
```

Comment:
Store the BanditVariationIndexer in the metadata cache because flagTimestamp is lonely